consensus/polybft/polybft_config.go

```diff
@@ -38,8 +38,10 @@ type PolyBFTConfig struct {
 	// Governance is the initial governance address
 	Governance types.Address `json:"governance"`
 
-	// MintableERC20Token denotes whether mintable ERC20 token is used
-	MintableERC20Token bool `json:"mintableERC20"`
+	// MintableNativeToken denotes whether mintable native token is used
+	MintableNativeToken bool `json:"mintableNative"`
+
+	NativTokenConfig *TokenConfig `json:"nativeTokenConfig"`
```

Comment:
Also consider adding a comment :wink: 
```suggestion
	NativeTokenConfig *TokenConfig `json:"nativeTokenConfig"`
```

Reply:
Fixed: https://github.com/0xPolygon/polygon-edge/pull/1376/commits/ab7a094805b4e5da93a395b0dd84f32cf7f251e8